.github/workflows/ci.yaml

```diff
@@ -0,0 +1,56 @@
+name: Ignition ros2 control CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup colcon workspace
+      id: configure
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        apt update -qq
+        apt install -qq -y lsb-release wget curl gnupg2
+        cd ..
+        mkdir -p /home/ros2_ws/src
+        cp -r ign_ros2_control /home/ros2_ws/src/
+        sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+        wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
+        sh -c 'echo "deb http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
+        curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
+        IGN_DEPS="libignition-gazebo5-dev libignition-plugin-dev libignition-utils1-cli-dev"
```

Comment:
You shouldn't need these Edifice dependencies here. `CMake` is defaulting to Citadel, which is being installed with `rosdep`.

```suggestion
```